xbmc/network/EventClient.cpp

```diff
@@ -370,7 +370,7 @@ bool CEventClient::OnPacketBUTTON(CEventPacket *packet)
   float famount = 0;
   bool active = (flags & PTB_DOWN) ? true : false;
   
-  CLog::Log(LOGDEBUG, "EventClient: button code %d %s", bcode, active ? "pressed" : "released");
+  CLog::Log(LOGDEBUG, "EventClient: button %s code %d %s", button, bcode, active ? "pressed" : "released");
```

Comment:
To avoid confusion we can log only the used parameters: 
```suggestion
  if (flags & PTB_USE_NAME)
    CLog::Log(LOGDEBUG, "EventClient: button name \"%s\" map \"%s\" %s",
              button.c_str(), map.c_str(), active ? "pressed" : "released");
  else
    CLog::Log(LOGDEBUG, "EventClient: button code %d %s", bcode, active ? "pressed" : "released");
```